share/spice/cryptocurrency/detail.handlebars

```diff
@@ -7,7 +7,7 @@
         <!-- LEFT INPUT SECTION -->
         <div class="frm clearfix">
             <div class="input__wrap input__wrap--left shadow">
-                <input class="frm__input frm__input--left" id="zci--cryptocurrency-amount-left" data-original="{{from-amount}}" data-processed="{{amount}}" value="{{amount}}">
+                <input class="frm__input frm__input--left" id="zci--cryptocurrency-amount-left" data-original="{{from-amount}}" data-processed="{{amount}}" value="{{amount}}" type="number" step="any">
```

Comment:
I remember now. The reason why it isn't `type="number"` is so that we can let the JS naturally display huge numbers in e-notation. Feedback shows that users actually liked this as the natural numbers got too big for the input fields.

Reply:
Ah, this was my initial suggestion, as I didn't know the reasoning behind not including it.

It is interesting that e-notation does not count as a number, imo it definitely should.

Reply:
> `type="number"` is so that we can let the JS naturally display huge numbers in e-notation.

I'm not sure I understand, isn't e-notation preferred? If so, we can display number in e-notation with `type="number"`. If you would prefer that, I can push a commit with that.

Reply:
Not all browsers.

Reply:
Ok, I would like to get this merged. Can you please remove this feature as it will cause issues and fix the merge conflict? ping @nalinbhardwaj